server/src/main/resources/routes

```diff
@@ -3,7 +3,10 @@
 # ~~~~
 
 # API routes
--> /api/v1  de.frosner.broccoli.ApiV1Router
+-> /api/v1  de.frosner.broccoli.routes.ApiV1Router
+
+# Downloads
+-> /downloads/ de.frosner.broccoli.routes.DownloadsRouter
```

Comment:
The endpoint can stay as it is very useful. What I'm saying is that it is not part of the UI. And anything that is not part of the UI, is part of the API, in my opinion. Then it makes more sense to add it as part of the API from a routing point of view, no?

I don't know how this endpoint is going to evolve but I see advantages of having a stable API and versioning it at the same time. This does not conflict IMO.

I didn't get the last part with the curls. If the dedicated API router is there why doesn't it return 404 if the API route does not exist?

@Gerrrr what do you think?